src/app/guides/how-to-get-funding-for-your-open-source-projects/how-to-get-funding-for-your-open-source-projects.mdx

```diff
@@ -0,0 +1,62 @@
+## How To Get Funding For Your Open Source Projects
+
+# Overview
+
+As wonderful as open source is, one of the biggest challenges maintained projects face is finding funding, especially ones that are sustainable.
+Now, don’t fret! Here are some suggestions on how to get funding for your open source projects as well as some legalities to consider in determining how to get these resources.
+
+## Direct Sponsorships
+
+Direct sponsorship is the most straightforward way to get funding for your project.
+When a user or a different company wants to sponsor your project, they send the funds right to your project’s financial account.
+Here is a list of resources that provide direct sponsorships:
+
+- **GitHub Sponsors**: This is a popular option that creators use to fund their projects.
+  It allows you to choose the number of tiers you charge and rewards that you can offer to potential sponsors.
+  It also provides you with the opportunity to support other developers and organizations, which can be a good way to network with others and gain more support.
+  If you’re interested in using this resource, check out the official website for more information: [https://github.com/sponsors](https://github.com/sponsors)
+- **Open Collective**: This is a company that is devoted to supporting and promoting open source projects.
+  Like GitHub sponsors, they allow you to choose the number of tiers you charge and reward you with potential sponsors.
+  In contrast, Open Source Collective provides information on how the money is being used.
+  If interested, check out their website for more information: [https://opencollective.com/how-it-works](https://opencollective.com/how-it-works)
+
+## Aggregated sponsorships
+
+In contrast to direct sponsorships, aggregated sponsorships are usually provided by a company or organization that can pool funds together.
+The benefits of this option are that you can get financial support from a larger pool of people on a regular basis, which can be a good way to keep your project funded.
```

Comment:
[Proofreading]

* It's both people and other types of entities, so the more general "donors" would be more correct I think
* Each of the things mentioned in this guide is a _"good way to keep your project funded"_, so I don't think we need to repeat that

```suggestion
The benefits of this option are that you can get financial support from a larger pool of donors.
```